config/settings.yml

```diff
@@ -6,6 +6,4 @@ google:
   maps_api_key: replace_me
 service_support:
   contact_email_address: becomingateacher@digital.education.gov.uk
-search_and_compare_ui:
-  base_url: http://localhost:5000
```

Comment:
Damn - I was deleting one that was added earlier and accidentally deleted this too :eyes:

Reply:
Oh no this was deleted before me @SamJamCul might be able to give insight as to why

Reply:
Ya, these are deleted because having a fallback may mean that we'll have the wrong configuration for an environment without realising it. By deleting it we're more likely to get an error and notice it.